lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -167,13 +176,49 @@
                 }
             ];
 
+            ordersHistoryData.unshift(
+                {
+                    'title': gettext('ORDER NAME'),
+                    'order_date': gettext('ORDER PLACED'),
+                    'price': gettext('TOTAL'),
+                    'number': gettext('INVOICE ID')
+                }
+            );
+
+            ordersSectionData = [
+                {
+                    title: gettext('Your Orders'),
+                    subtitle: gettext(
+                        'View details of our past purchases and, potentially request a refund for a past order. For ' +
+                        'more information on refund request eligibility, please read here.'
+                    ),
+                    fields: _.map(ordersHistoryData, function(order) {
+                        orderNumber = order.number;
+                        if (orderNumber === 'INVOICE ID') {
+                            orderNumber = 'invoice-id';
+                        }
+                        return {
+                            'view': new AccountSettingsFieldViews.OrderHistoryFieldView({
+                                title: order.title,
+                                total_price: order.price,
+                                invoice_id: order.number,
```

Comment:
for consistency across UI and code I would suggest changing the variable name from `invoice_id` to `orderId`